easybuild/easyconfigs/b/BgeeDB/BgeeDB-2.26.0-foss-2021a-R-4.1.0.eb

```
@@ -0,0 +1,66 @@
+easyblock = 'Bundle'
+
+name = 'BgeeDB'
+version = '2.26.0'
+versionsuffix = '-R-%(rver)s'
+
+# This library does not need last Bioconductor dependencies
+# but better to have its last version from the last Bioconductor version
+local_bioc_version = 3.17
+
+homepage = 'https://github.com/BgeeDB/BgeeDB_R'
+description = """Annotation and gene expression data retrieval from Bgee database.
+TopAnat, an anatomical entities Enrichment Analysis tool for UBERON ontology."""
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('R',          '4.1.0'),
+]
+
+dependencies = [
+    ('R-bundle-Bioconductor', '3.13', versionsuffix),
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://bioconductor.org/packages/%(local_bioc_version)s/bioc/src/contrib/',
+        'https://bioconductor.org/packages/%(local_bioc_version)s/bioc/src/contrib/Archive/%(name)s',
+        'https://bioconductor.org/packages/%(local_bioc_version)s/data/annotation/src/contrib/',
+        'https://bioconductor.org/packages/%(local_bioc_version)s/data/experiment/src/contrib/',
```

```suggestion
        'https://bioconductor.org/packages/%s/bioc/src/contrib/' % local_bioc_version,
        'https://bioconductor.org/packages/%s/bioc/src/contrib/Archive/%%(name)s' % local_bioc_version,
        'https://bioconductor.org/packages/%s/data/annotation/src/contrib/' % local_bioc_version,
        'https://bioconductor.org/packages/%s/data/experiment/src/contrib/' % local_bioc_version,
```

Local variables do not get replaced, so have to be used as per the suggested replacement instead.

---

So in the test report we see it trying to download with the `%(local_bioc_version)s` inthe URL:

```
== 2023-09-08 09:55:32,644 filetools.py:832 WARNING URL https://bioconductor.org/packages/%(local_bioc_version)s/bioc/src/contrib/topGO_2.52.0.tar.gz was not found (HTTP response code 400), not trying again
```